README.md

```diff
@@ -51,10 +51,10 @@ Looking for help? Check out the
 * **LTS**: Releases that receive Long Term Support, with a focus on stability
   and security. Every even-numbered major version will become an LTS release.
   LTS releases receive 12 months of _Active LTS_ support and a further 18 months
-  of _Maintenance_. LTS release lines have alphabetically-ordered code names,
+  of _Maintenance_. LTS release lines have alphabetically ordered code names,
   beginning with v4 Argon. There are no breaking changes or feature additions,
   except in some special circumstances.
-* **Nightly**: Code from the Current branch built every 24-hours when there are
+* **Nightly**: Code from the Current branch is built every 24 hours when there are
   changes. Use with caution.
```

Comment:
The Nightly build is made from "code from the Current branch built every 24 hours". I think adding "is" is not correct here.
```suggestion
* **Nightly**: Code from the Current branch built every 24 hours when there are
  changes. Use with caution.
```

We might be able to rephrase this to be clearer and more consistent from one bullet point to the next, however I don't have a suggestion.